py-polars/tests/unit/io/test_scan.py

```diff
@@ -929,3 +930,30 @@ def test_predicate_stats_eval_nested_binary() -> None:
         ),
         pl.DataFrame({"x": [2]}),
     )
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("streaming", [True, False])
+def test_scan_csv_bytesio_memory_usage(
+    streaming: bool,
+    memory_usage_without_pyarrow: MemoryUsage,
+) -> None:
+    memory_usage = memory_usage_without_pyarrow
+
+    # Create CSV that is ~70-85 MB in size:
+    f = io.BytesIO()
```

Comment:
Can we test this with a little bit less data. I assume it is noticeable with less.

Reply:
Done.